client/packages/system/src/Patient/CreatePatientModal/PatientResultsTab.tsx

```diff
@@ -12,12 +12,39 @@ import { usePatient } from '../api';
 import { Gender, useCreatePatientStore } from '../hooks';
 import { PatientFragment } from '../api/operations.generated';
 
+function genderToGenderInput(gender: Gender): GenderInput {
```

Comment:
:nit we are using ES6 syntax throughout and I'd prefer that to the `function` syntax.
( there's a couple of exceptions due to typescript constraints.. apart from that.. )

```
const genderToGenderInput = (gender: Gender): GenderInput => {
  switch (gender) {
    case Gender.MALE:
      return GenderInput.Male;
    case Gender.FEMALE:
      return GenderInput.Female;
    case Gender.TRANSGENDER_MALE:
      return GenderInput.TransgenderMale;
    case Gender.TRANSGENDER_FEMALE:
      return GenderInput.TransgenderFemale;
    case Gender.NON_BINARY:
      return GenderInput.NonBinary;
    case Gender.UNKNOWN:
    default:
      return GenderInput.Unknown;
  }
};
```

Reply:
Ok done. (I tend to choose functions because there are slightly easier to write, but then I also use arrow functions inconsistently, so arrow functions it is)